packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.service.js

```diff
@@ -169,6 +169,10 @@ export default class PciStoragesColdArchiveService {
       );
   }
 
+  getArchiveEndpoint(serviceName, regionName) {
```

Comment:
**nitpick:** function name is not meaningful as it gets the region info.

Reply:
I changed it to "getArchiveRegionDetails()".